examples/browser-to-browser/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "js-libp2p-webrtc-browser-to-server",
+  "version": "1.0.0",
+  "description": "Connect a browser to a server",
+  "type": "module",
+  "scripts": {
+    "start": "vite",
+    "build": "vite build",
+    "go-relay": "cd ../go-libp2p-server && go run ./relay",
+    "node-relay": "node relay.js",
+    "test:firefox": "npm run build && playwright test --browser=firefox tests",
+    "test:chrome": "npm run build && playwright test tests",
```

Comment:
We should add webkit, but this will fail until
https://github.com/libp2p/js-libp2p/pull/1627 is fixed.

Reply:
Is this a blocker @MarcoPolo or a new issue to work once https://github.com/libp2p/js-libp2p/pull/1627 is completed?

Reply:
Not a blocker